modules/@apostrophecms/i18n/ui/apos/components/AposI18nLocalize.vue

```diff
@@ -84,55 +86,63 @@
                 class="apos-locales-filter"
                 ref="searchInput"
               />
-              <transition-group tag="ul" name="selected-list" class="apos-selected-locales">
+              <transition-group
+                tag="ul"
+                name="selected-list"
+                class="apos-selected-locales"
+              >
                 <li
-                  v-for="locale in selectedLocales"
-                  :key="locale.name"
+                  v-for="loc in selectedLocales"
+                  :key="loc.name"
```

Comment:
why changing `locale` to `loc` (which is less clear, I think)

Reply:
because there is a higher scope variable with the same name, was a lint error not my choice.